os/include/tinyara/sched.h

```diff
@@ -620,7 +620,6 @@ struct tcb_s {
 
 	uint32_t uheap;			/* User heap object pointer */
 #ifdef CONFIG_APP_BINARY_SEPARATION
-	uint32_t ram_start;		/* Start address of RAM partition for this app */
 	uint32_t ram_size;		/* Size of RAM partition for this app */
```

Comment:
We could also remove ram_size variable.

Reply:
Yes, I agree. I will update with another PR for checking `ram_size` and `ramstart`.